templates/includes/npm.php

```diff
@@ -16,36 +16,36 @@
 <p>
   <strong>Note: Unlike most of the other Enable JavaScript modules, you cannot load this one as an old-school ES4
   JavaScript library.</strong>
-  This is because it tests for browser features (in this case, the <code>&lt;dialog&gt;</code> tag) and if the browser
+  This is because it tests for browser features (in this case, the <code>&lt;dialog&gt;</code> tag), and if the browser
   doesn't support it, load the polyfill using<a
     href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_import"> the ES6
     <code>import()</code> function</a>.
 </p>
 <?php } else { ?>
 
-<p>If you haven't done so already, choosing which you should use is obviously a major architectural decision.
+<p>If you haven't done so already, choosing which you should use is a major architectural decision.
   Here are a few articles that will help you decide:
 </p>
 
 
 <ul>
   <li><a href="https://gist.github.com/jkrems">Jan Olaf Krems</a> gives a great overview of the
-    <a href="https://gist.github.com/jkrems/b14894e0b8efde10aa10a28c652d3541">JavaScript File Format Differences</a>
+    <a href="https://gist.github.com/jkrems/b14894e0b8efde10aa10a28c652d3541">JavaScript File format differences.</a>
   </li>
   <li><a href="https://ecmascript.engineer/">Joe Honton</a> discusses that <a
-      href="https://betterprogramming.pub/2020-004-the-rollout-of-modules-is-complete-d25f04870284">With ES Modules and
+      href="https://betterprogramming.pub/2020-004-the-rollout-of-modules-is-complete-d25f04870284">with ES Modules and
       HTTP/2, You May Not Need Webpack Anymore</a>
```

Comment:
If you're not capitalizing some of the article title in order to just include it as text, then it should all be lowercase:

```suggestion
      href="https://betterprogramming.pub/2020-004-the-rollout-of-modules-is-complete-d25f04870284">with ES modules and
      HTTP/2, you may not need Webpack anymore</a>
```